assets/hubert/.gitignore

```diff
@@ -1,2 +1,3 @@
 *
 !.gitignore
+!hubert_inputs.pth
```

Comment:
I don't think it is necessary to raise these files as they are generated

Reply:
This is necessary when using jit to generate the model for the first time, as you cannot predict when users will use jit.
Of course, hubert_inputs.pth is unnecessary as we did not use it, you can delete it。